malloc-trace/inc/com/centreon/malloc_trace/orphan_container.hh

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright 2024 Centreon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ */
+
+#ifndef CMT_ORPHAN_CONTAINER_HH
+#define CMT_ORPHAN_CONTAINER_HH
+
+#include <com/centreon/common/node_allocator.hh>
+
+#include "funct_info_cache.hh"
+
+namespace com::centreon::malloc_trace {
+
+constexpr size_t max_backtrace_size = 10;
+
+/**
+ * @brief information or free or malloc with stacktrace
```

Comment:
I don't understand this.
When I see the constructor, I can see funct_name. So it can be "malloc", "free" or something else??

Maybe examples could be useful...